ga/src/main/java/nl/tudelft/serg/evosql/fixture/type/DBTypeSelector.java

```diff
@@ -43,21 +43,21 @@ public DBType create(int dataType, int length) {
             case Types.LONGNVARCHAR:
                 return new DBString(EvoSQLConfiguration.MAX_STRING_LENGTH, "LONGNVARCHAR");
             case Types.CHAR:
-                return new DBString(length, "CHAR");
+                return new DBString(length, "CHAR(" + length + ")");
             case Types.NCHAR:
                 return new DBString(length, "NCHAR");
             // Boolean values
             case Types.BIT:
-                return new DBBoolean(); // HSQLDB doesn't support bit
-            case Types.BOOLEAN:
-                return new DBBoolean("BOOLEAN");
+                return new DBBoolean("BOOLEAN"); // HSQLDB doesn't support bit
             // Dates and times
             case Types.DATE:
                 return new DBDate();
             case Types.TIME:
                 return new DBTime();
             case Types.TIMESTAMP:
                 return new DBDateTime("TIMESTAMP");
+            case Types.NUMERIC:
```

Comment:
This hotfix should also be undone I guess, unless we can make it 'official'.

Reply:
Well the parameters are specific for the `erpnext` schema so I don't think it should be merged. However, maybe we can set a more 'normal' default value. The `ga` does not really care what the size is for a type as long as it matches the schema. Perhaps there is even a way to pass these parameters along to the `TypeSelector`. My idea: for now we remove the hotfixes and we open an issue where we explore the possibilities of extending the functionality for `DBType` variables.

Reply:
Yeah an issue would be a good thing.